src/VisualStudio/IntegrationTest/IntegrationTests/VisualStudioIntegrationTests.csproj

```diff
@@ -28,6 +28,7 @@
     <Compile Include="CSharp\CSharpIntelliSense.cs" />
     <Compile Include="CSharp\CSharpInteractive.cs" />
     <Compile Include="AbstractEditorTest.cs" />
+    <Compile Include="CSharp\CSharpSignatureHelp.cs" />
```

Comment:
Does this match what we had in the existing XML based test?  Should that file be deleted as part of this?